src/System.Management.Automation/engine/Modules/ImportModuleCommand.cs

```diff
@@ -1890,6 +1879,26 @@ protected override void ProcessRecord()
             }
         }
 
+        internal override IList<PSModuleInfo> ImportModulesUsingWinCompat(IEnumerable<string> moduleNames, IEnumerable<ModuleSpecification> moduleFullyQualifiedNames, ImportModuleOptions importModuleOptions)
+        {
+            IList<PSModuleInfo> moduleProxyList = new List<PSModuleInfo>();
+            var WindowsPowerShellCompatRemotingSession = CreateWindowsPowerShellCompatResources();
+            if (WindowsPowerShellCompatRemotingSession != null)
```

Comment:
If you invert this condition, you can have a simple return followed by the real main logic of the method

Reply:
Updated.